### DISCUSSION TITLE
Data Privacy and Security

### DISCUSSION THREAD

**Discussion by ragbh7** [NHA-ABDM/nhcx#7] [Q&A] [answered]
Does NHCX access or read any of the data transferred via its platform?

**Comment by vishi24** [comment, marked as answer]
Data transferred through NHCX is consist of two parts
1. Header - base 64 encoded
2. Request body - The body is encrypted with the receiver's public key, can only be decrypted by the corresponding private key.

Only the protected header data encoded in base64 is accessed and read by NHCX.
